spec.md

```diff
@@ -242,6 +245,7 @@ help intermediate gateways determine how to route the events.
   [Versioning of Attributes in the Primer](primer.md#versioning-of-attributes)
   for more information.
 * Constraints:
+  * MUST be a URL [RFC 3986](https://tools.ietf.org/html/rfc3986#section-1.1.3)
```

Comment:
I'd advise you to keep this PR focused on the eventID and open a separate PR for this line, if you want to push it.

(If you do, I'd also suggest to add `URL` to the type system instead of awkwardly constraining the `URI-reference` type - or alternatively change MUST to RECOMMEND.)

Reply:
Will open a new PR for this, if I still care about it by the time I've applied your other comments :)